stable/ibex-dashboard/templates/storageclass.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClassName) -}}
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+ name: "{{ .Values.persistence.storageClass }}"
+provisioner: {{ .Values.persistence.storage.provisioner }}
+parameters:
+ skuName: {{ .Values.persistence.storage.parameters.skuName }}
```

Comment:
@itye-msft One of my comments must have been removed… I'll recap:

Is this chart only applicable when run on Azure? I ask because `skuName` will only work on `kubernetes.io/azure-disk`.

Reply:
@scottrigby I missed the extra whitespace, my bad. I have committed the fix.

This is strange, I responded to your comment about Azure, but it seems my comment disappeared as well.
The application is not bounded to to any Azure specific resources and can run anywhere, including locally or any other cloud, private or public of your choice. It's basically a Node.js web app.
I have been testing it on Azure and locally, and other people and companies are using it successfully so far. 
The `skuName` is just a working example, and it will work with any PVC of your choice.

Reply:
@scottrigby Thank you, that's brilliant. 
I changed the template to accept any amount of params dynamically from the values.yaml file. 

This chart persists the user "dashboards" in files, this is why the "dashboards" files folder is mounted outside of the pod. (in case the pod is replaced, the data is persisted). 
From this perspective, I think the template would be a good fit.